composer.json

```diff
@@ -10,6 +10,7 @@
     "xp-forge/frontend": "^7.0 | ^6.0",
     "xp-forge/handlebars": "^10.0 | ^9.3",
     "xp-forge/yaml": "^9.0 | ^8.0 | ^7.0 | ^6.0",
+    "xp-forge/json": "^6.0 | ^5.0",
```

Comment:
This does not add a new dependency, `xp-forge/json` was already required by `xp-forge/frontend`.